src/components/KMenu/KMenu.vue

```diff
@@ -92,32 +92,32 @@ const proceed = (): void => {
 @import '@/styles/functions';
 
 .k-menu {
-  background-color: var(--white);
-  border: 1px solid var(--grey-300);
-  border-radius: 4px;
-  padding-bottom: 8px;
-  padding-top: 11px;
+  background-color: var(--white, var(--kui-color-background, $kui-color-background));
+  border: var(--kui-border-width-10, $kui-border-width-10) solid var(--grey-300, var(--kui-color-background-neutral-weak, $kui-color-background-neutral-weak));
```

Comment:
issue: `color-background` token in `border` property

Reply:
fixed to use `kui-color-border-neutral-weak` instead